src/store/__test__/main.spec.ts

```diff
@@ -1111,6 +1114,49 @@ describe('Main store', () => {
     expect(res).toBeTruthy();
   });
 
+  it('should accept search query and return results based on query ', async () => {
+    uiStore.searchText = 'test';
+    const mockApiResponse = {
+      status: 200,
+      ok: true,
+      json: (): Promise<any> => Promise.resolve([mockBounties[0]])
+    };
+    fetchStub.resolves(Promise.resolve(mockApiResponse));
+    const store = new MainStore();
+    await store.setPersonBounties([mockBounties[0].bounty]);
+    const bounties = await store.getPeopleBounties({
+      resetPage: false,
+      search: uiStore.searchText,
+      limit: 10,
+      page: 1,
+      sortBy: 'created'
+    });
+    expect(bounties).toHaveLength(1);
+    expect(bounties[0].body.title).toEqual(uiStore.searchText);
+  });
+
+  it('should return filter by languages, status response', async () => {

```

Comment:
@elraphty is this test not useful for the frontend we're just mocking the API response

Reply:
Done